src/spaceObjects/cpuShip.cpp

```diff
@@ -158,7 +158,7 @@ void CpuShip::orderRoamingAt(sf::Vector2f position)
     orders = AI_Roaming;
     order_target = NULL;
     order_target_location = position;
-    this->addBroadcast(FVF_Friendly, "Searching for hostiles around " + string(position.x) + "," + string(position.y) + ".");
+    this->addBroadcast(FVF_Friendly, tr("cpulog", "Searching for hostiles around {posx},").format({{"posx", string(position.x)}}) + string(position.y) + ".");
```

Comment:
This looks off:
`tr("cpulog", "Searching for hostiles around {x}, {y}").format({{"x", string(position.x)}, {"y", string(position.y)}})`